### PR TITLE
Prettifies README

### PR DESCRIPTION
**Pull request by sanchezfdezjavier**
# Description

Prettifies readme.

# Checklist:

- [x] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that validate the behaviour of the software
- [x] I validated that new and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ghislainbourgeois — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
-# magma-access-gateway-snap
+<h1 align="center">
+  <img src="magma-logo-purple.svg" width="500" ></a>
+  <p align="center"><a href="https://snapcraft.io/magma-access-gateway"><img src="https://snapcraft.io/magma-access-gateway/badge.svg" alt="access gateway badge"/></p>
+  <br/>
+  Magma Access Gateway Snap
+</h1>
 
-## Description
+[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/terraform)
```
Comment (Gmerold) [inline README.md:8]:
```suggestion
[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/magma-access-gateway)
```

File: README.md [hunk 2]
```diff
@@ -17,35 +26,34 @@ For more information on Magma please visit the [official website](https://magmac
 > :warning: Installing this snap will affect your computer's networking configuration.
 > Make sure it is installed on designated hardware (personal computers are strongly discouraged).
 
-### System requirements
+## System requirements
 
-**Hardware (baremetal strongly recommended)**
+### Hardware (baremetal strongly recommended)
```
Comment (Gmerold) [inline README.md:31]: For some reason this doesn't render properly.

**Review by Gmerold — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sanchezfdezjavier) [reply README.md]: Oh, fixed.

File: README.md (hunk 2 above)
Comment (sanchezfdezjavier) [reply README.md]: I think it renders now, let me know if it works for you.

**Review by Gmerold — APPROVED** [review]

**Pull request merged**